Jenkinsfile

```diff
@@ -490,6 +506,107 @@ az aks create                      \
             }
         }
     }
+*/
+
+    //def eksVersions = ["1.10", "1.11"]
+    def eksVersions = ["1.11"]
```

Comment:
reminder to enable tests for other supported k8s versions

Reply:
Yes

Reply:
Done.

Reply:
i guess reverted :slightly_smiling_face: